lib/src/index.ts

```diff
@@ -87,18 +88,61 @@ const executeCommand = async (
       }
     }
   } catch (error) {
-    const eventId = Sentry.captureException(error);
-    const eventStr = `\n\nError ID: ${logger.info(eventId)}`;
-
     if (process.env.DEBUG === "true") {
       console.error(logger.info("Development stack trace:\n"), error);
     }
 
-    return await quit(
-      logger.errorText(
-        "Something went wrong. Please contact support or try again later."
-      ) + eventStr
-    );
+    let sentryOptions = undefined;
+    let exitCode = undefined;
+    let errorText =
+      "Something went wrong. Please contact support or try again later.";
+
+    if (isDittoError(error)) {
+      sentryOptions = {
+        extra: { message: error.message, ...(error.data || {}) },
+      };
+      exitCode = error.exitCode;
+
+      if (isDittoErrorType(error, ErrorType.ConfigYamlLoadError)) {
+        errorText = error.message;
+      } else if (isDittoErrorType(error, ErrorType.ConfigParseError)) {
+        const invalidKeys = Array.from(
+          new Set(
+            error.data.issues
+              .map((issue) => ("keys" in issue ? issue.keys : null))
+              .flat()
+              .filter(Boolean)
+          )
+        );
```

Comment:
```suggestion
const invalidKeys = Array.from(
          new Set(
            error.data.issues
              .map((issue) =>
                "keys" in issue
                  ? `${issue.path.join(".")}${
                      issue.path.length > 0 ? "." : ""
                    }${issue.keys}`
                  : null
              )
              .flat()
              .filter(Boolean)
          )
        );
```

Reply:
doing this will help identify if the issue is from a nested field or a top level field.